examples/Payment/Create.php

```diff
@@ -0,0 +1,157 @@
+// In case you want to test in your local machine first, set runtime enviroment to LOCAL
+MercadoPagoConfig::setRuntimeEnviroment(MercadoPagoConfig::LOCAL);
+
+// Step 3: Initialize the API client
+$client = new PaymentClient();
+
+try {
+    // Step 4: Create the request array
+    $request = [
+        "transaction_amount" => 1000,
+        "description"        => "Título do produto",
+        "payment_method_id"  => "master",
+        "token"              => "{{CARD_TOKEN}}",
+        "installments"       => 1,
+        "application_fee"    => null,
+        "binary_mode"        => true,
+        "capture"            => true,
+        "external_reference" => "Pedido01",
+        "metadata"           => new \stdClass(),
+        "notification_url"   => "http://requestbin.fullcontact.com/1ogudgk1",
+        "sponsor_id"         => null,
+        "statement_descriptor" => "LOJA 123",
+        "payer" => [
+            "first_name" => "Nome",
+            "last_name"  => "Sobrenome",
+            "email"      => "{{EMAIL}}",
+            "identification" => [
+                "number" => "{{DOCUMENT_NUMBER}}",
+                "type"   => "CPF"
+            ],
+            "phone" => [
+                "area_code" => "11",
+                "number"    => "{{PHONE_NUMBER}}"
+            ],
+            "address" => [
+                "street_name"    => "Av. das Nações Unidas",
+                "street_number"  => "3003",
+                "zip_code"       => "206233-2002"
+            ]
+        ],
+        "forward_data" => [
+            "sub_merchant" => [
+                "sub_merchant_id"    => "1234183712",
+                "mcc"                => "5462",
+                "country"            => "BRA",
+                "address_door_number"=> 123,
+                "zip"                => "2222222",
+                "document_number"    => "222222222222222",
+                "city"               => "SÃO PAULO",
+                "address_street"     => "RUA A",
+                "legal_name"         => "LOJINHA DO ZÉ",
+                "region_code_iso"    => "BR-MG",
+                "region_code"        => "BR",
+                "document_type"      => "CNPJ",
+                "phone"              => "{{PHONE}}",
+                "url"                => "www.rappi.com.br"
+            ]
+        ],
+        "additional_info" => [
+            "items" => [
+                [
+                    "id"          => "1941",
+                    "title"       => "25/08/2022 | Pista Inteira5 lote - GREEN VALLEY GRAMADO 2022",
+                    "description" => "25/08/2022 | Pista Inteira5 lote - GREEN VALLEY GRAMADO 2022",
+                    "category_id"  => "Tickets",
+                    "quantity"     => 1,
+                    "unit_price"   => 1000.00,
+                    "event_date"   => "2019-12-25T19:30:00.000-03:00",
+                    "picture_url"  => "<URL_IMAGE>",
```

Comment:
[req] Manter padronização do placeholder com uso de chaves `{}`
```suggestion
                    "picture_url"  => "{{IMAGE_URL}}",
```